src/llama_recipes/data/llama_guard/README.md

```diff
@@ -0,0 +1,98 @@
+# Finetuning Data Formatter
+
+The finetuning_data_formatter script provides classes and methods for formatting training data for finetuning a language model on a specific task. The main classes are:
```

Comment:
"a language model" -> Llama Guard

Reply:
This isn't for all LMs we have

Reply:
Quick question, if someone wants to train llama guard from scratch on top of llama 2 7B, can they use this formatter as well?